docs/UserGuide.md

```diff
@@ -91,20 +86,65 @@ You can use the sample data pre-loaded in the application to get play around and
 
 </div>
 
-### Viewing help : `help`
+#### **2.2.3 Types of User Input**
+To expand.
 
-* Gives instructions on how to enter orders into the CLI.
-* Displays a message with a list of all available commands and their format.
-* To return to the main order list click on the `Return to the main order list` button. 
-* Help can also be accessed by clicking the `help` button in the top left corner of the application or by clicking the `F1` keyboard key.
+##### `NAME`
+* The name of the person who has made the order.
```

Comment:
I think this can be better phrased as "The name of the person who placed the order."

Reply:
sounds good

Reply:
maybe we should mention customers in our UG, not sure if that's more or less confusing...?

e.g. The name of the customer who placed the order."